.github/scripts/pr-find-issues.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -e
+
+PR="$1"
+REPO="$2"
+
+if [ "$REPO" == "" ]; then
+  REPO="keycloak/keycloak"
+fi
```

Comment:
This can be shortened a bit by using [shell parameter expansion](https://tiswww.case.edu/php/chet/bash/bashref.html#Shell-Parameter-Expansion):

```suggestion
REPO=${2:-"keycloak/keycloak"}
```

Reply:
Sure it's shorter, but also significantly less readable.